avocado/utils/kernel.py

```diff
@@ -64,11 +64,19 @@ def __repr__(self):
 
     @property
     def vmlinux(self):
-        vmlinux_path = os.path.join(self.linux_dir, 'vmlinux')
+        if not self.build_dir:
+            return None
+        vmlinux_path = os.path.join(self.build_dir, 'vmlinux')
         if os.path.isfile(vmlinux_path):
             return vmlinux_path
         return None
 
+    @property
+    def build_dir(self):
```

Comment:
If `build_dir` is the same as an existing `linux_dir`, shouldn't one of them be removed?  Maybe turn `linux_dir` into a `_build_dir` and have the `build_dir` property to return it only if the directory exists, similar to the `vmlinux` property?

Reply:
If I understood correctly in the past the build dir was different from source dir (i.e. `linux_dir`). At some point someone made `build_dir` == `linux_dir` and forgot to delete build_dir. I agree that in current implementation it would makes sense to turn `linux_dir` in `build_dir`. So yes, I will change this in v2.